diff --git "a/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\as-is\\03-listar-productos.txt" "b/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\to-be\\03-listar-productos.txt"
index b94c871..b6df7b1 100644
--- "a/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\as-is\\03-listar-productos.txt"
+++ "b/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\to-be\\03-listar-productos.txt"
@@ -11,6 +11,7 @@ ALERTA: stock mínimo de Dolex
 ALERTA: stock mínimo de Amoxicilina
 ALERTA: stock mínimo de Loratadina
 ALERTA: stock mínimo de Diclofenaco
+ALERTA: stock mínimo de CremaHidratante
 ALERTA: Dolex próximo a vencer
 ALERTA: Ibuprofeno próximo a vencer
 ALERTA: Amoxicilina próximo a vencer
@@ -21,6 +22,7 @@ ALERTA: VitaminaC próximo a vencer
 ALERTA: Diclofenaco próximo a vencer
 ALERTA: Aspirina próximo a vencer
 ALERTA: Naproxeno próximo a vencer
+ALERTA: Helado próximo a vencer
 
 ==============================
       SISTEMA FARMACIA
@@ -48,6 +50,12 @@ VitaminaC		15	6000
 Diclofenaco		3	11000
 Aspirina		9	7500
 Naproxeno		7	13500
+ProtectorSolar		6	25000
+CremaHidratante		2	18000
+Gaseosa		30	3500
+AguaBotella		40	2000
+Helado		8	4500
+Snack		12	2500
 
 ==============================
       SISTEMA FARMACIA
