diff --git "a/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\as-is\\04-listar-clientes.txt" "b/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\to-be\\04-listar-clientes.txt"
index 01b6379..db40c6d 100644
--- "a/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\as-is\\04-listar-clientes.txt"
+++ "b/C:\\Users\\david\\Documents\\202602\\Arq\\Parcial1\\Reto2\\codigo\\caracterizacion\\reto2\\salidas\\sc1\\to-be\\04-listar-clientes.txt"
@@ -11,6 +11,7 @@ ALERTA: stock mínimo de Dolex
 ALERTA: stock mínimo de Amoxicilina
 ALERTA: stock mínimo de Loratadina
 ALERTA: stock mínimo de Diclofenaco
+ALERTA: stock mínimo de CremaHidratante
 ALERTA: Dolex próximo a vencer
 ALERTA: Ibuprofeno próximo a vencer
 ALERTA: Amoxicilina próximo a vencer
@@ -21,6 +22,7 @@ ALERTA: VitaminaC próximo a vencer
 ALERTA: Diclofenaco próximo a vencer
 ALERTA: Aspirina próximo a vencer
 ALERTA: Naproxeno próximo a vencer
+ALERTA: Helado próximo a vencer
 
 ==============================
       SISTEMA FARMACIA
